docs: Add info about auto-signoff (#8915)
keptn/keptn: CONTRIBUTING.md

git rebase master
git push --set-upstream origin feature/123/foo
```

> Make sure you sign-off your commits. To do this automatically check [this](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#auto-signoff-commit-messages). Finally, go to GitHub and create a Pull Request. There should be a PR template already prepared for you.

Reviewer: ```suggestion
Make sure you sign off your commits. To do this automatically check [this](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md#auto-signoff-commit-messages). Finally, go to GitHub and create a Pull Request. There should be a PR template already prepared for you.
```

- secret-service
- shipyard-controller
- statistics-service

### Auto signoff commit messages

We have a DCO check which runs on every PR to check if the commit has been signed off.

> you can do 
> ```bash
> git commit -ammend --signoff
> ```
> 
> or something like
> ```bash
> git rebase HEAD~2 --signoff
> ```

Reviewer: ```suggestion
You can do 
```bash
git commit --amend --signoff
```

or something like
```bash
git rebase HEAD~2 --signoff
```
```

> to signoff commits, but it is inconvenient. You can automate this process.

Reviewer: ```suggestion
to sign off commits, but it is inconvenient. You can automate this process.
```

> To automatically signoff commits  : 

Reviewer: ```suggestion
To automatically sign off commits:
```
